src/adapter/src/catalog/builtin.rs

```diff
@@ -1070,7 +1070,7 @@ pub static MZ_COLUMNS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
     desc: RelationDesc::empty()
         .with_column("id", ScalarType::String.nullable(false))
         .with_column("name", ScalarType::String.nullable(false))
-        .with_column("position", ScalarType::Int64.nullable(false))
+        .with_column("position", ScalarType::UInt64.nullable(false))
```

Comment:
I'm actually on the fence for all of these position columns since they are used in pg views and need to be casted.